### PR TITLE
add taxiZoneExemption credential type to amsterdam issuer

### PR DESCRIPTION
**Pull request by ams-azaaiman**
Credential type taxiZoneExemption toegevoegd binnen de amsterdam issuer.
Tevens de bestaande keys van deze issuer uitgebreid om ook voor revocation te kunnen gebruiken.
Daarna een sign van het scheme gedaan.

### CODE REVIEW

File: amsterdam/Issues/taxizoneOntheffing/description.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <Name>
+                <en>Exemption valid from</en>
+                <nl>Ontheffing geldig vanaf</nl>
+            </Name>
+            <Description>
+                <en>Your taxi zone exemption is valid from this date</en>
+                <nl>Je taxizone-ontheffing is geldig vanaf deze datum</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="geldigTotEnMet" optional="true">
```
Comment (ivard) [inline amsterdam/Issues/taxizoneOntheffing/description.xml:49]: The attribute identifiers should be in English. That's our guideline in pbdf and it's best practice to keep it consistent here.

File: amsterdam/Issues/taxizoneOntheffing/description.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <Name>
+                <en>Exemption number</en>
+                <nl>Ontheffingsnummer</nl>
+            </Name>
+            <Description>
+                <en>Exemption number</en>
+                <nl>Ontheffingsnummer</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="geldigVanaf" optional="true">
```
Comment (ivard) [inline amsterdam/Issues/taxizoneOntheffing/description.xml:39]: Are these dates always optional?

**Review by ivard — CHANGES_REQUESTED** [review]

File: amsterdam/Issues/taxizoneOntheffing/description.xml (hunk 1 above)
Comment (ams-azaaiman) [reply amsterdam/Issues/taxizoneOntheffing/description.xml]: Changed it, also the credentialType itself has been renamed to be in English.

Might be good to add this to the issuer documentation as well.

File: amsterdam/Issues/taxiZoneExemption/description.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <nl>Demo Taxizone-ontheffing Amsterdam</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>amsterdam</IssuerID>
+    <CredentialID>taxiZoneExemption</CredentialID>
+    <Description>
+        <en>Taxi zone exemption Amsterdam</en>
+        <nl>Taxizone-ontheffing Amsterdam</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
```
Comment (ivard) [inline amsterdam/Issues/taxiZoneExemption/description.xml:17]: Are you sure you want to have this if you also have a `validFrom` and a `validUntil`? This flag enforces that you cannot have two credentials of this instance at the same time in your IRMA app.

Comment (ams-azaaiman) [reply amsterdam/Issues/taxiZoneExemption/description.xml:17]: For the time being we want this.
The `validFrom` and `validUntil` are for now just for display / informational purposes.
For the future we do expect to drop the singleton requirement, but that might not happen for another year.
For your information, this exemption will typically be valid for 2 years.

Or would it cause issues if we change the credentialType to not be a singleton in the future?

Comment (ivard) [reply amsterdam/Issues/taxiZoneExemption/description.xml:17]: It should be possible to change this later if I'm right. I thought, if you know it for sure now that it's needed, then you can also set it correctly now. But if you want to think this a bit more through, then that's also fine and then we keep it `true` for now.

**Review by ivard — APPROVED** [review]

**Review by sanderhollaar — APPROVED** [review]

**Pull request merged**